saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -269,24 +273,9 @@ def _commit_changes(cls, info, instance, cleaned_input, new_instance):
         )
 
     @classmethod
-    def _refresh_lines_unit_price(cls, info, instance, cleaned_input, new_instance):
-        if new_instance:
-            # It is a new instance, all new lines have already updated prices.
-            return
-        shipping_address = cleaned_input.get("shipping_address")
-        if shipping_address and instance.is_shipping_required():
-            update_order_prices(
-                instance,
-                info.context.plugins,
-                info.context.site.settings.include_taxes_in_prices,
-            )
-        billing_address = cleaned_input.get("billing_address")
-        if billing_address and not instance.is_shipping_required():
-            update_order_prices(
-                instance,
-                info.context.plugins,
-                info.context.site.settings.include_taxes_in_prices,
-            )
+    def should_invalidate_prices(cls, instance, cleaned_input, new_instance) -> bool:
+        # Force price recalculation for all new instances
+        return new_instance
```

Comment:
Could we add typing here or rename the `new_instance` attribute to something like `is_new_instance`? I was quite sure that the `new_instance` is an `Order` instance, not a boolean value.